### PR TITLE
imply rewind correctly what it does

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: cmd/du-main.go [hunk 1]
```diff
@@ -43,11 +43,11 @@ var (
 		},
 		cli.StringFlag{
 			Name:  "rewind",
-			Usage: "move back in time",
+			Usage: "include object(s) versions at specified date in past, to be used wth `--versions`",
```
Comment (klauspost) [inline cmd/du-main.go:46]:
```suggestion
			Usage: "include all object versions no later than specified date, to be used with `--versions`",
```

File: cmd/du-main.go [hunk 2]
```diff
@@ -43,11 +43,11 @@ var (
...
 		},
 		cli.BoolFlag{
 			Name:  "versions",
-			Usage: "include all objects versions",
+			Usage: "include all object(s) versions",
```
Comment (klauspost) [inline cmd/du-main.go:50]:
```suggestion
			Usage: "include all object versions",
```

File: cmd/ls-main.go [hunk 3]
```diff
@@ -34,11 +34,11 @@ var (
 	lsFlags = []cli.Flag{
 		cli.StringFlag{
 			Name:  "rewind",
-			Usage: "back in time",
+			Usage: "list object(s) versions at specified date in past, to be used wth `--versions`",
```
Comment (klauspost) [inline cmd/ls-main.go:37]:
```suggestion
			Usage: "list object versions no later than specified date, to be used with `--versions`",
```

File: docs/minio-client-complete-guide.md [hunk 4]
```diff
@@ -889,19 +890,20 @@ USAGE:
...
-  --force                       allow a recursive remove operation
-  --dangerous                   allow site-wide removal of objects
-  --incomplete, -I              remove incomplete uploads
-  --fake                        perform a fake remove operation
-  --stdin                       read object names from STDIN
-  --older-than value            remove objects older than L days, M hours and N minutes LMN[d|h|m]. (default: 0)
-  --newer-than value            remove objects newer than L days, M hours and N minutes LMN[d|h|m]. (default: 0)
-  --encrypt-key value           encrypt/decrypt objects (using server-side encryption with customer provided keys)
-  --help, -h                    show help
+  --versions                       remove object(s) and all its version(s)
```
Comment (klauspost) [inline docs/minio-client-complete-guide.md:893]:
```suggestion
  --versions                       remove object(s) and all its versions
```

File: docs/minio-client-complete-guide.md [hunk 5]
```diff
@@ -889,19 +890,20 @@ USAGE:
...
+  --rewind value                   rollback object(s) to older version(s)
```
Comment (klauspost) [inline docs/minio-client-complete-guide.md:894]:
```suggestion
  --rewind value                   roll back object(s) to current version at specified time
```

**Review by klauspost — CHANGES_REQUESTED** [review]
A few grammatical tweaks.

File: cmd/du-main.go (hunk 1 above)
Comment (vadmeste) [reply cmd/du-main.go:46]: 'to be used' is actually 'can be used'

File: docs/minio-client-complete-guide.md (hunk 4 above)
Comment (harshavardhana) [reply docs/minio-client-complete-guide.md:893]: It is on purpose `--versions` lists `a version` or `versions`

Comment (klauspost) [reply docs/minio-client-complete-guide.md:893]: Yeah, but ` remove object(s) and all its version` doesn't make sense grammatically.

**Review by klauspost — APPROVED** [review]
👍

**Review by vadmeste — COMMENTED** [review]
Thanks!

LGTM

**Pull request merged**